controllers/admin/AdminCmsContentController.php

```diff
@@ -75,14 +78,6 @@ public static function getCurrentCMSCategory()
         return self::$category;
     }
 
-    public function viewAccess($disable = false)
```

Comment:
i want :) `$this->admin_cms_categories->tabAccess` and `$this->admin_cms->tabAccess` are defined in `__construct` and there is no need for `viewAccess`